Common/UI/Hotbar/NewHotbar.cs

```diff
@@ -379,6 +380,14 @@ private static void DrawSkill(SpriteBatch spriteBatch, float off, float opacity,
 			{
 				skillCombatPlayer.HotbarSkills[skillIndex] = null;
 			}
+
+			if (Main.mouseMiddle && Main.mouseMiddleRelease && !SmartUiLoader.GetUiState<TreeState>().Visible && skill.Tree is not null)
+			{
+				TreeState tree = SmartUiLoader.GetUiState<TreeState>();
```

Comment:
You have to set Release booleans to false to consume the clicks.
```suggestion
			if (Main.mouseMiddle && Main.mouseMiddleRelease && !SmartUiLoader.GetUiState<TreeState>().Visible && skill.Tree is not null)
			{
				Main.mouseMiddleRelease = false;
				TreeState tree = SmartUiLoader.GetUiState<TreeState>();
```